public/app/percona/shared/components/PerconaBootstrapper/PerconaUpdateVersion/PerconaUpdateVersion.test.tsx

```diff
@@ -13,7 +12,6 @@ import { StoreState } from 'app/types';
 import { EnhancedStore } from '@reduxjs/toolkit';
 
 const checkUpdatesChangeLogsSpy = jest.spyOn(GrafanaUpdates, 'checkUpdatesChangeLogs');
-const fetchUserDetailsActionSpy = jest.spyOn(UserTourReducer, 'fetchUserDetailsAction');
 
 describe('PerconaUpdateVersion', () => {
```

Comment:
I think a test case for the snooze functionality (that the button calls the api) would be helpful.